integration-tests/gatsby-pipeline/__tests__/lazy-image-build/develop.js

```diff
@@ -0,0 +1,162 @@
+const { execFileSync } = require(`child_process`)
+const execa = require(`execa`)
+const path = require(`path`)
+const fs = require(`fs-extra`)
+const basePath = path.resolve(__dirname, `../../`)
+const request = require(`request-promise-native`)
+
+// 2 min
+jest.setTimeout(2000 * 60)
+
+const bootDevelop = () =>
+  new Promise(resolve => {
+    const devProcess = execa(`yarn`, [`develop`], {
```

Comment:
You can also use the package `start-server-and-test` I believe, e.g.

`start-server-and-test develop http://localhost:8000`

We use it pretty heavily on the e2e-tests.

Reply:
yeah I saw that but I didn't really see how this would work inside a test when I want to control what's coming next.